.github/workflows/lint-prose.yml

```diff
@@ -0,0 +1,25 @@
+name: Lint prose
+
+on:
+  push:
+    branches-ignore:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
I was going to suggest something like paths even for builds. Because we don't need to be rebuilding docs everytime a component changes, nor rebuilding components for docs. But I can update this one for sure too. Would probably also include md files in other paths, as they can be included in docs and linting.